batch/batch/front_end/front_end.py

```diff
@@ -671,18 +695,59 @@ def check_service_account_permissions(user, sa):
     raise web.HTTPBadRequest(reason=f'unauthorized service account {(sa["namespace"], sa["name"])} for user {user}')
 
 
+# this route is deprecated
+# use "/api/v1alpha/batches/{batch_id}/updates/{update_id}/jobs/create" instead
 @routes.post('/api/v1alpha/batches/{batch_id}/jobs/create')
 @rest_authenticated_users_only
-async def create_jobs(request: aiohttp.web.Request, userdata: dict):
+async def create_jobs(request: web.Request, userdata: dict):
     app = request.app
 
     if app['frozen']:
         log.info('ignoring batch create request; batch is frozen')
         raise web.HTTPServiceUnavailable()
 
     batch_id = int(request.match_info['batch_id'])
+    user = userdata['username']
+    db: Database = app['db']
+
+    records = [
+        r
+        async for r in db.select_and_fetchall(
+            '''
+SELECT batch_updates.update_id FROM batches
+INNER JOIN batch_updates
+ON batches.id = batch_updates.batch_id
+WHERE batches.user = %s AND batches.id = %s;
+''',
+            (user, batch_id),
+        )
+    ]
+
+    if not records:
+        raise web.HTTPNotFound()
+
+    if len(records) > 1:
+        raise web.HTTPBadRequest(reason='Use /api/v1alpha/batches/{batch_id}/updates/{update_id}/jobs/create instead')
```

Comment:
I think this message should have a bit more context. Maybe: "Updating an extant batch can only be done with the updates endpoint, not this endpoint.".